### PR TITLE
Add Rom and Shahar to the list of committers

### PR DESCRIPTION
**Pull request by shahar1**

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]

File: landing-pages/site/content/en/announcements/_index.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# July 19, 2024
```
Comment (eladkal) [inline landing-pages/site/content/en/announcements/_index.md:16]: Should be the date of the mailing list announcement

**Comment by eladkal** [comment]
We are missing also adding to list in
https://airflow.apache.org/community/
Maybe the step is missing from the guide?

**Comment by shahar1** [comment]
> We are missing also adding to list in https://airflow.apache.org/community/ Maybe the step is missing from the guide?

I think that it draws the info from `landing-pages/site/data/committers.json` :)

**Comment by eladkal** [comment]
Ah right

**Review by eladkal — APPROVED** [review]

**Pull request merged**